Project.toml

```diff
@@ -9,29 +9,29 @@ DeepDiffs = "ab62b9b5-e342-54a8-a765-a90f495de1a6"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
-ImageInTerminal = "d8c32880-2388-543b-8c61-d9f865259254"
 LazyModules = "8cdb02fc-e678-4876-92c5-9defec4f444e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+XTermColors = "c8c2cc18-de81-4e68-b407-38a3a0c0491f"
 
 [compat]
 BSON = "0.3"
-Colors = "0.10, 0.11, 0.12"
+Colors = "0.10 - 0.12"
 CSVFiles = "1"
-DataFrames = "0.21, 0.22"
+DataFrames = "0.21 - 0.22"
 DeepDiffs = "1.1"
-Distances = "0.7, 0.8, 0.9, 0.10"
+Distances = "0.7 - 0.10"
```

Comment:
This doesn't matter much, but generally, a good PR (patchset) should only consist of necessary changes with no stylish changes (people have different flavors).

Reply:
I thinks it's more readable. This syntax is only possible with recent julia versions (> 1.6 is ok).

Reply:
It's an unrelated enhancement though.
And that's the problem.

PRs should do exactly one thing.
Doing more than 1 makes for problems.

It makes review harder. As there is more things to consider at once.
And can delay getting the PR merged, since we might bog down into a debate over if its better or not.
Where as in two PRs one could be quickly merged and the other could wait.

It also makes isolating bugs and reverting harder.
And can lead to more merge conflicts